### DISCUSSION TITLE
Module 3 is launched NOW!!!

### DISCUSSION THREAD

**Discussion by chjch** [SERVIR-WA/GALUP#77] [Announcements]
Hello GALUP community,

We just launched [Module 3](https://github.com/SERVIR-WA/GALUP/blob/master/training/1_lu/modules/module3.md). Please check it out and let us know if you have questions.

Cheers,

Changjie

**Comment by Achidago** [comment]
Hi Dr. Chen,

I'm experiencing the error below on the soil condition model.

"Traceback (most recent call last):
File "C:\Users\silac\AppData\Roaming\QGIS\QGIS3\profiles\default\processing\scripts\weighted_sum.py", line 128, in processAlgorithm
input_gdf[output_clm] = utils.weighted_sum(input_gdf, fields_weights)
File "C:\Users\silac\AppData\Roaming\Python\Python37\site-packages\pylusat\utils.py", line 223, in weighted_sum
return df[[*col_weights]].dot(wgt_df)
File "C:\PROGRA~1\QGIS3~1.10\apps\Python37\lib\site-packages\pandas\core\frame.py", line 1140, in dot
np.dot(lvals, rvals), index=left.index, columns=other.columns
File "<__array_function__ internals>", line 6, in dot
TypeError: unsupported operand type(s) for *: 'NoneType' and 'float'

Error encountered while running Weighted Sum of Fields(Soil Condition Weighted)
Error encountered while running Weighted Sum of Fields(Soil Condition Weighted)
Execution failed after 24.23 seconds

Loading resulting layers
The following layers were not correctly generated.<ul><li>C:\Users\silac\AppData\Local\Temp/processing_0d24879674a34c75bded9fef252434c8/db7e0e07486d46fcaf18bfeeb6f993e0/WeightedSum.gpkg</li></ul>You can check the 'Log Messages Panel' in QGIS main window to find more information about the execution of the algorithm."

Your assistance will be appreciated.
@chjch
